licenses/CDLA-Permissive-2.0

```diff
@@ -0,0 +1,35 @@
+Community Data License Agreement - Permissive - Version 2.0
+
+This is the Community Data License Agreement - Permissive, Version 2.0 (the "agreement"). Data Provider(s) and Data Recipient(s) agree as follows:
```

Comment:
For better readability, please rewrap these long lines to a reasonable length (72 characters is what we often use, but anything 70-ish is fine).

Reply:
Updated and added to the group!